src/context/Theme.tsx

```diff
@@ -149,14 +158,24 @@ export function SettingsProvider({ children }: PropsWithChildren<{}>) {
     const root = document.documentElement
     root.style.setProperty("--cards-border-width", `${context.borderWidthCards}px`)
     root.style.setProperty("--cards-shadow", `${context.shadowCards}px`)
-    root.style.setProperty("--cards-gap", `${context.spaceBetweenCards}px`)
+    root.style.setProperty("--cards-gap", `${context.spaceBetweenCards}px`) 
   }, [context])
 
+  useEffect(() => {
```

Comment:
I'm not sure I understand the full-extent of this logic, but if I do I think it is not necessary and can be moved to the initial `useClientEffect`. Maybe i'm missing something though

Reply:
I removed this entirly. We only compute the "hidden" state from the message stored in the store. The state is false by default. I think its cleaner this way